### ISSUE TITLE
Size of git repo and  Examples

### ISSUE THREAD

**Issue by Yaraslaut** [paulxshen/Luminescent.jl#36]
Hi, thanks for the amazing work
I wanted to bring up a couple of things:
*  I noticed the repository size is currently 300M with 140M in .git, so if you are not planning to squash all commits and lose git history it might become an  issue. 
* Also, i wanted to play around with the package a bit, and found out that examples aren't working. Do you have a rough estimate of when the API might become more stable so I can dive in ?

**Comment by paulxshen** [comment]
You're right I'll condense git at some point. Current focus is on our python api gdsfactory [plugin](https://paulxshen.github.io/Luminescent.jl/luminescent.html) for photonics.  If you're doing RF, I'll need couple wks to revamp the Julia API

**Comment by Yaraslaut** [comment]
> You're right I'll condense git at some point. Current focus is on our python api gdsfactory [plugin](https://paulxshen.github.io/Luminescent.jl/luminescent.html) for photonics. If you're doing RF, I'll need couple wks to revamp the Julia API

Alright, good to know :) 
If you already have an idea about Julia API  that you want to have, let me know,  I can make a PR to bring it faster

**Comment by paulxshen** [comment]
Cool - what kinds of simulations are you running - what kinds of sources?

**Comment by Yaraslaut** [comment]
> Cool - what kinds of simulations are you running - what kinds of sources?

I am mainly interested in julia fdtd implementation to include custom material equations

**Comment by paulxshen** [comment]
as in nonlinear materials?

**Comment by Yaraslaut** [comment]
yes

**Comment by paulxshen** [comment]
in update.jl you can edit:
```
    dEdt = (∇ × H - σ * E - J) / ϵ
...
    dHdt = -(∇ × E + σm * H) / μ
```
I hacked up julia operators to do this :)
